packages/teleport/src/services/agents/types.ts

```diff
@@ -18,9 +18,15 @@ import { App } from 'teleport/services/apps';
 import { Database } from 'teleport/services/databases';
 import { Node } from 'teleport/services/nodes';
 import { Kube } from 'teleport/services/kube';
-import { Desktop } from 'teleport/services/desktops';
+import { Desktop, WindowsDesktopService } from 'teleport/services/desktops';
 
-export type AgentKind = App | Database | Node | Kube | Desktop;
+export type AgentKind =
+  | App
+  | Database
+  | Node
+  | Kube
+  | Desktop
```

Comment:
Technically `Desktop` isn't an agent. What's the purpose of having both desktop and desktop service in here? Is it because you wait for desktops to be reported in that last step?

Reply:
`Desktop` was already there, and is used by the response type of `DesktopService.fetchDesktops` (`AgentResponse<Desktop>`, with the constraint `AgentResponse<T = AgentKind>`). This is how the existing desktops page grabs the list of desktops.

My addition is `DesktopService` as I wanted to use the same response type for `DesktopService.fetchDesktopServices` (`AgentResponse<DesktopService>`).